.circleci/config.yml

```diff
@@ -3,12 +3,26 @@ jobs:
   build:
     working_directory: "~/website"
     docker:
-      - image: circleci/golang:1.11
+      - image: felicianotech/docker-hugo:0.47.1
     steps:
       - checkout
-      - run: 
+      - run:
           name: install misspell
           command: curl -L -o ./install-misspell.sh https://git.io/misspell && sh ./install-misspell.sh
       - run:
           name: Check Spelling
           command: ./bin/misspell -error ~/website/content/
+      - deploy:
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+              hugo
+              cd docs
+              git config --global user.email "sparkle_pony_2000@qri.io"
+              git config --global user.name "Qri CI Deployer"
+              git init
```

Comment:
I'm slightly confused how this works. You're creating a new repo here, but then pushing to the gh-pages branch that exists on github. So the /docs folder is removed from the master branch, but still exists in the gh-pages branch that gets deployed, right? And the gh-pages branch will always be a single commit with no older history? Not saying this is a problem, just trying to make sure I understand how this is working.

Reply:
Correct, but to clarify, there is no `/docs` directory in `gh-pages`, rather `gh-pages` is the _contents_ of `/doc` after the site is built.

`gh-pages` is force pushed, so it always has only the last successful build.

Reply:
Cool, sounds good.